jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -232,9 +244,30 @@ def vue_do_aper_phot(self, *args, **kwargs):
                     d['id'] = 1
                     self.app._aper_phot_results = _qtable_from_dict(d)
 
+            # Radial profile
+            if plt is None:
+                self.radial_plot = 'radial plot N/A: install matplotlib'
+            else:
+                reg_bb = reg.bounding_box
+                reg_ogrid = np.ogrid[reg_bb.iymin:reg_bb.iymax, reg_bb.ixmin:reg_bb.ixmax]
+                radial_dx = reg_ogrid[1] - reg.center.x
+                radial_dy = reg_ogrid[0] - reg.center.y
+                radial_r = np.hypot(radial_dx, radial_dy).ravel() * u.pix
+                radial_img = comp_no_bg_cutout.ravel()
+                with quantity_support():
+                    fig, ax = plt.subplots()
+                    ax.plot(radial_r, radial_img, 'k.')

```

Comment:
@larrybradley , I hope I got this right. Would be nice if you double check if I am using `regions` correctly for the radial plot. Thanks!